### ISSUE TITLE
initPaymentSheet style params is always automatic on Android

### ISSUE THREAD

**Issue by AlexSirenko** [stripe/stripe-react-native#692]
**Describe the bug**
initPaymentSheet style params is always automatic on Android

**To Reproduce**
Create initPaymentSheet with different style param (alwaysLight/alwaysDark), check that light or dark on Android depends on system theme only.

**Expected behavior**
Param style affects visual colors on android.

**Smartphone (please complete the following information):**
- Android 11

**Comment by tomparkp** [comment]
We're facing this issue as well as #709 which prevent us from utilizing this on a heavily stylized app our company is planning on launching soon. These and more customization options for the style of the payment sheet would be a big help for us, we're considering abandoning this library in favor of a custom UI.

**Comment by charliecruzan-stripe** [comment]
Hi, unfortunately we're blocked on this until https://github.com/stripe/stripe-android/issues/4229 😞 Sorry for the wait. But as stated in that issue, you could force the theme for your whole app with:

```
// force dark
AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
// force light 
AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
```

You could call this in your `MainApplication`

**Comment by charliecruzan-stripe** [comment]
closing as a duplicate of https://github.com/stripe/stripe-react-native/issues/285
